File: Languages/Day06/Answer.ts
//question1
const yourName: string = "Bob";
let yourAge: number = 22;
const buildWebApps: boolean = true;

console.log(yourName, yourAge, buildWebApps);

//question2
const passwordLength: number = 8;

if (passwordLength < 8) {
    console.log("Password too short")
}
else {
    console.log("Password length okay")
}

//question3
const tasks: string[] = ["Welcome to the store", "How can I help you?", "I'm looking for a mask"]

for (const task of tasks) {
    console.log(task);
}

//question4
function addTax(price: number, taxRate: number): number {
    return price + (price * taxRate);
}

let result = addTax(100, .06)
console.log(result)

//question5
const status: string = "outside";

function showStatus(): void {
    const status: string = "inside"
    console.log(status);
}

showStatus()
console.log(status)

File: Languages/Day06/answer.R
#question 1

name <- "bob"
age <- 86
build_web_apps <- FALSE

cat (name, age, build_web_apps, "\n")

#question 2

password_length <- 33

if(password_length < 8){
  print("Password too short")
}
else {
  print("Password length okay")
}

#question 3

tasks <- c("make a movie", "wash the car", "pick up laundry")

for (x in tasks){
  print(x)
}

#question 4

add_tax <- function (price, tax_rate){
  return (price + (price * tax_rate))
}

finalPrice <- add_tax(88, .06)

print (finalPrice)

#question 5

status <- "outside"

show_status <- function(){
  status <- "inside"
  print(status)
}

show_status()
print(status)